pkg/reconciler/revision/resources/queue.go

```diff
@@ -158,7 +157,7 @@ func getResourcePercentageFromAnnotations(m map[string]string, k string) (bool,
 	if err != nil {
 		return false, 0
 	}
-	return true, float32(value)
+	return true, float32(value / 100)
```

Comment:
Since this was not caught by any test a test for this should be added, I think.

Reply:
"resources percentage in annotations" handles that